polkadot/erasure-coding/Cargo.toml

```diff
@@ -12,14 +12,15 @@ workspace = true
 [dependencies]
 polkadot-primitives = { path = "../primitives" }
 polkadot-node-primitives = { package = "polkadot-node-primitives", path = "../node/primitives" }
-novelpoly = { package = "reed-solomon-novelpoly", version = "1.0.0" }
-parity-scale-codec = { version = "3.6.1", default-features = false, features = ["derive", "std"] }
+novelpoly = { package = "reed-solomon-novelpoly", git = "https://github.com/paritytech/reed-solomon-novelpoly.git" }
```

Comment:
TODO: switch to a new release that includes https://github.com/paritytech/reed-solomon-novelpoly/pull/36

Reply:
done